inference-engine/include/ie/gpu/gpu_context_api_dx.hpp

```diff
@@ -145,25 +147,24 @@ static inline Blob::Ptr make_shared_blob_nv12(size_t height, size_t width, Remot
 }
 
 /**
- * @brief This function is used to obtain remote context object from ID3D11Device
+ * @brief This function is used to obtain remote context object from
+ * ID3D11Device
  * @param core Inference Engine Core object instance
  * @param deviceName A name of to create a remote context for
  * @param device A pointer to ID3D11Device to be used to create a remote context
  * @return A shared remote context instance
  */
 static inline D3DContext::Ptr make_shared_context(Core& core, std::string deviceName, ID3D11Device* device) {
-    ParamMap contextParams = {
-        { GPU_PARAM_KEY(CONTEXT_TYPE), GPU_PARAM_VALUE(VA_SHARED) },
```

Comment:
Can we keep this as is?

Reply:
I didn't find any option to do it, because it really depends on the place. I disabled clang-format for this place and return the original format